DBImport/InsertData.py

```diff
@@ -0,0 +1,117 @@
+        The database object
+
+    """
+    env_path = Path('.') / '.env'
+    load_dotenv(dotenv_path=env_path)
+    username, password = os.getenv('Mongo_User'), os.getenv('Mongo_Password')
+    db_name = os.getenv('Mongo_DBName')
+    client = MongoClient('mongodb+srv://' + username + ':' + password
+                         + os.getenv('Mongo_Postfix'))
+    return client.get_database(db_name)
+
+
+def check_file_open(filename):
+    """Check if the json file exist in the specific directory.
+
+    Args:
+        filename: the path to the desired file
+    Raises:
+        FileNotFoundError: Maybe try to find your file?
+    Returns:
+        The database object
+
+    """
+    if filename:
+        with open(filename, 'r') as f:
+            return json.load(f)
+    else:
+        logging.info('File name: ', filename, ' cannot be found!')
+        raise Exception('File not found')
+
+
+def insert_data(course_list, dept_list, quarter_name):
+    """Insert data into databse.
+
+    Get every single Course and Department object from the lists and 
+    insert them into the correct database collections 
+        (named quarter_name + 'course'/'departments')
+
+    Args:
+        course_list, dept_list: the lists of Course/Department objects
+    Raises:
+        pymongo.errors: possibly connection errors or conficuration errors
+    Returns:
+        None
+ 
+    """
+    db = get_db()
+    course_collection = db[quarter_name + ' courses']
+    dept_collection = db[quarter_name + ' departments']
+
+    for course in course_list:
+        temp_course = vars(course)
+        course_collection.insert_one(temp_course)
+    for dept in dept_list:
+        temp_dept = vars(dept)
+        dept_collection.insert_one(temp_dept)
+
+
+def main():
+    """Runner of this DBImport program.
+
+    With help of other functions, this main function could read the data from
+    json files and put them into desired databses.
+    """
+    env_path = Path('.') / '.env'
+    load_dotenv(dotenv_path=env_path)
+    logging.info('Excecution Started At: ', datetime.datetime.now())
+    config = ConfigParser()
+    config.read(os.getenv('Config_File_Name'))
+    path = config['locations']['path']
+    year = config['data_info']['start_year']
+
+    try:
+        while(config['locations'][str(year)]):
+            all_quarters_in_year = config['locations'][str(year)].split(',')
+            for each_quarter in all_quarters_in_year:
+                quarter_name, filename = each_quarter[:-16].replace('_', ' '), path + each_quarter
+                course_raw_data = check_file_open(filename)
+                course_list, department_list = from_raw_to_list(course_raw_data, quarter_name)
+                logging.info(datetime.datetime.now(),
+                             ': Total loaded course for quarter ', each_quarter, ' ', len(course_list))
+                insert_data(course_list, department_list, quarter_name)
+            year += 1
+    except Exception as e:
```

Comment:
since `e` is not use.
`except:` should be ok?

Reply:
would logging.info(e) be good?

Reply:
then that should be `logging.error(e)`